content/it/mutual-transport-layer-security.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Mutual Transport Layer Security (mTLS)
+status: Completed
+category: Concetto
+tags: ["sicurezza", "networking", ""]
+---
+
+## Cos'è
+
+Il Mutual TLS (mTLS) è una tecnica utilizzata per autenticare e criptare i messaggi inviati tra due [servizi](/it/service/).
+Il Mutual TLS è il protocollo [Transport Layer Security](/it/transport-layer-security/) (TLS) standard,
+ma invece di validare l'identità di una sola connessione entrambi i lati sono validati.
+
+## Quali problematiche affronta
+
+I [microservizi](/it/microservices/) comunicano tramite una rete e,

```

Comment:
Eviterei di andare a capo dopo la virgola

Reply:
è soltanto un modo per tenere corte le righe in Markdown e favorire i cambiamenti singoli, lato front-end viene tenuto tutto nella stessa riga